[7.x] [Fleet] Rename `CONFIG_CHANGE` action to `POLICY_CHANGE` (#78221)

jen-huang opened the pull request:
Backports the following commits to 7.x:
 - [Fleet] Rename `CONFIG_CHANGE` action to `POLICY_CHANGE` (#78221)

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/)
* Commit: f3357b8c53685f003f164b1dd6dfd17c7a263fd5
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute xpack-kibana-ciGroup7](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/execution/node/634/log)
* [Execute xpack-kibana-ciGroup7](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/execution/node/761/log)

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/testReport)

<details><summary>X-Pack EPM API Integration Tests.x-pack/test/ingest_manager_api_integration/apis/fleet/agents/complete_flow·ts.Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/testReport//junit/X-Pack%20EPM%20API%20Integration%20Tests/x-pack_test_ingest_manager_api_integration_apis_fleet_agents_complete_flow·ts/Ingest_Manager_Endpoints_Fleet_Endpoints_fleet_agent_flow_should_work)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://github.com/elastic/kibana/issues/75241

[00:00:00]       │
[00:00:00]         └-: Ingest Manager Endpoints
[00:00:00]           └-> "before all" hook
[00:00:08]           └-: Fleet Endpoints
[00:00:08]             └-> "before all" hook
[00:01:36]             └-: fleet_agent_flow
[00:01:36]               └-> "before all" hook
[00:01:36]               └-> "before all" hook
[00:01:36]                 │ info [empty_kibana] Loading "mappings.json"
[00:01:36]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_1/coLz9uO0RtaT_aSmCn-ovw] deleting index
[00:01:36]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/Lxe_rM5IQviaP0ids4C7xQ] deleting index
[00:01:36]                 │ info [empty_kibana] Deleted existing index [".kibana_2",".kibana_1"]
[00:01:36]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana] creating index, cause [api], templates [], shards [1]/[1]
[00:01:36]                 │ info [empty_kibana] Created index ".kibana"
[00:01:36]                 │ debg [empty_kibana] ".kibana" settings {"index":{"number_of_replicas":"1","number_of_shards":"1"}}
[00:01:36]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana/U81NbBYjR-2PNNRRreDbzw] update_mapping [_doc]
[00:01:36]                 │ debg Migrating saved objects
[00:01:36]                 │ proc [kibana]   log   [17:25:14.755] [info][savedobjects-service] Creating index .kibana_2.
[00:01:36]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2] creating index, cause [api], templates [], shards [1]/[1]
[00:01:36]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.kibana_2]
[00:01:36]                 │ proc [kibana]   log   [17:25:14.825] [info][savedobjects-service] Reindexing .kibana to .kibana_1
[00:01:36]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_1] creating index, cause [api], templates [], shards [1]/[1]
[00:01:36]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.kibana_1]
[00:01:36]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.tasks] creating index, cause [auto(task api)], templates [], shards [1]/[1]
[00:01:36]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.tasks]
[00:01:36]                 │ info [o.e.t.LoggingTaskListener] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] 5822 finished with response BulkByScrollResponse[took=2ms,timed_out=false,sliceId=null,updated=0,created=0,deleted=0,batches=0,versionConflicts=0,noops=0,retries=0,throttledUntil=0s,bulk_failures=[],search_failures=[]]
[00:01:36]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana/U81NbBYjR-2PNNRRreDbzw] deleting index
[00:01:36]                 │ proc [kibana]   log   [17:25:15.189] [info][savedobjects-service] Migrating .kibana_1 saved objects to .kibana_2
[00:01:36]                 │ proc [kibana]   log   [17:25:15.194] [info][savedobjects-service] Pointing alias .kibana to .kibana_2.
[00:01:36]                 │ proc [kibana]   log   [17:25:15.219] [info][savedobjects-service] Finished in 465ms.
[00:01:36]               └-> "before all" hook
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38862, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38860, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38870, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38868, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:36]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38878, url.original: /package/system/0.5.3
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38876, url.original: /package/endpoint/0.13.1
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38886, url.original: /package/system/0.5.3/
[00:01:36]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38884, url.original: /package/endpoint/0.13.1/
[00:01:36]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38898, url.original: /search?kibana.version=7.10.0
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38896, url.original: /search?kibana.version=7.10.0
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38906, url.original: /package/endpoint/0.13.1
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38904, url.original: /package/system/0.5.3
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38916, url.original: /package/system/0.5.3
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38918, url.original: /package/endpoint/0.13.1/
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38912, url.original: /package/endpoint/0.13.1
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38922, url.original: /package/system/0.5.3/
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38928, url.original: /package/system/0.5.3/
[00:01:37]                 │ info [docker:registry] 2020/09/29 17:25:15 source.ip: 172.17.0.1:38930, url.original: /package/endpoint/0.13.1/
[00:01:37]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:38]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:38]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:38]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:20 source.ip: 172.17.0.1:38948, url.original: /package/system/0.5.3
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:20 source.ip: 172.17.0.1:38946, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:20 source.ip: 172.17.0.1:38952, url.original: /package/system/0.5.3/
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:21 source.ip: 172.17.0.1:38960, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:21 source.ip: 172.17.0.1:38962, url.original: /package/system/0.5.3
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:21 source.ip: 172.17.0.1:38966, url.original: /package/system/0.5.3/
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:21 source.ip: 172.17.0.1:38970, url.original: /package/system/0.5.3
[00:01:42]                 │ info [docker:registry] 2020/09/29 17:25:21 source.ip: 172.17.0.1:38974, url.original: /package/system/0.5.3/
[00:01:42]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:43]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:44]                 │ info [docker:registry] 2020/09/29 17:25:22 source.ip: 172.17.0.1:38986, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:44]                 │ info [docker:registry] 2020/09/29 17:25:22 source.ip: 172.17.0.1:38984, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:44]                 │ info [o.e.x.s.a.r.TransportPutRoleAction] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updated role [fleet_enroll]
[00:01:44]                 │ info [o.e.x.s.a.u.TransportPutUserAction] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updated user [fleet_enroll]
[00:01:46]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:46]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:47]               └-> should work
[00:01:47]                 └-> "before each" hook: global before each
[00:01:47]                 └-> "before each" hook: beforeSetupWithDockerRegistry
[00:01:47]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/4cZVMsvYQ8GoCc0WbbE38g] update_mapping [_doc]
[00:01:49]                 └- ✖ fail: Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work
[00:01:49]                 │      Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
[00:01:49]                 │       at Assertion.assert (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:100:11)
[00:01:49]                 │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:227:8)
[00:01:49]                 │       at Assertion.be (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:69:22)
[00:01:49]                 │       at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
[00:01:49]                 │ 
[00:01:49]                 │ 

```

#### Stack Trace
```
Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
    at Assertion.assert (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
```
</details>

---

<details><summary>X-Pack EPM API Integration Tests.x-pack/test/ingest_manager_api_integration/apis/fleet/agents/complete_flow·ts.Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/testReport//junit/X-Pack%20EPM%20API%20Integration%20Tests/x-pack_test_ingest_manager_api_integration_apis_fleet_agents_complete_flow·ts/Ingest_Manager_Endpoints_Fleet_Endpoints_fleet_agent_flow_should_work_2)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://github.com/elastic/kibana/issues/75241

[00:00:00]       │
[00:00:00]         └-: Ingest Manager Endpoints
[00:00:00]           └-> "before all" hook
[00:00:09]           └-: Fleet Endpoints
[00:00:09]             └-> "before all" hook
[00:01:38]             └-: fleet_agent_flow
[00:01:38]               └-> "before all" hook
[00:01:38]               └-> "before all" hook
[00:01:38]                 │ info [empty_kibana] Loading "mappings.json"
[00:01:38]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/NQhWbC-nQDqIE-tLEeNV_g] deleting index
[00:01:38]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_1/O604twa_Rf-0iDtsTJJzEQ] deleting index
[00:01:38]                 │ info [empty_kibana] Deleted existing index [".kibana_2",".kibana_1"]
[00:01:38]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana] creating index, cause [api], templates [], shards [1]/[1]
[00:01:38]                 │ info [empty_kibana] Created index ".kibana"
[00:01:38]                 │ debg [empty_kibana] ".kibana" settings {"index":{"number_of_replicas":"1","number_of_shards":"1"}}
[00:01:38]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana/GKJ-ILFOSGG1QwUxJikIew] update_mapping [_doc]
[00:01:38]                 │ debg Migrating saved objects
[00:01:38]                 │ proc [kibana]   log   [16:37:23.552] [info][savedobjects-service] Creating index .kibana_2.
[00:01:38]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2] creating index, cause [api], templates [], shards [1]/[1]
[00:01:38]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.kibana_2]
[00:01:38]                 │ proc [kibana]   log   [16:37:23.619] [info][savedobjects-service] Reindexing .kibana to .kibana_1
[00:01:38]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_1] creating index, cause [api], templates [], shards [1]/[1]
[00:01:38]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.kibana_1]
[00:01:38]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.tasks] creating index, cause [auto(task api)], templates [], shards [1]/[1]
[00:01:38]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updating number_of_replicas to [0] for indices [.tasks]
[00:01:38]                 │ info [o.e.t.LoggingTaskListener] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] 5846 finished with response BulkByScrollResponse[took=2.1ms,timed_out=false,sliceId=null,updated=0,created=0,deleted=0,batches=0,versionConflicts=0,noops=0,retries=0,throttledUntil=0s,bulk_failures=[],search_failures=[]]
[00:01:38]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana/GKJ-ILFOSGG1QwUxJikIew] deleting index
[00:01:38]                 │ proc [kibana]   log   [16:37:23.981] [info][savedobjects-service] Migrating .kibana_1 saved objects to .kibana_2
[00:01:38]                 │ proc [kibana]   log   [16:37:23.986] [info][savedobjects-service] Pointing alias .kibana to .kibana_2.
[00:01:38]                 │ proc [kibana]   log   [16:37:24.012] [info][savedobjects-service] Finished in 463ms.
[00:01:38]               └-> "before all" hook
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48820, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48818, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48828, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48826, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:38]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48836, url.original: /package/endpoint/0.13.1
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48834, url.original: /package/system/0.5.3
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48842, url.original: /package/system/0.5.3/
[00:01:38]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48844, url.original: /package/endpoint/0.13.1/
[00:01:39]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48862, url.original: /search?kibana.version=7.10.0
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48864, url.original: /search?kibana.version=7.10.0
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48874, url.original: /package/endpoint/0.13.1
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48876, url.original: /package/endpoint/0.13.1
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48878, url.original: /package/system/0.5.3
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48880, url.original: /package/system/0.5.3
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48890, url.original: /package/endpoint/0.13.1/
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48892, url.original: /package/endpoint/0.13.1/
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48888, url.original: /package/system/0.5.3/
[00:01:39]                 │ info [docker:registry] 2020/09/29 16:37:24 source.ip: 172.17.0.1:48896, url.original: /package/system/0.5.3/
[00:01:39]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:40]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:40]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:40]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:43]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49044, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49042, url.original: /package/system/0.5.3
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49048, url.original: /package/system/0.5.3/
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49060, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49062, url.original: /package/system/0.5.3
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49066, url.original: /package/system/0.5.3/
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49072, url.original: /package/system/0.5.3
[00:01:44]                 │ info [docker:registry] 2020/09/29 16:37:29 source.ip: 172.17.0.1:49076, url.original: /package/system/0.5.3/
[00:01:45]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:46]                 │ info [docker:registry] 2020/09/29 16:37:31 source.ip: 172.17.0.1:49140, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:46]                 │ info [docker:registry] 2020/09/29 16:37:31 source.ip: 172.17.0.1:49138, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:46]                 │ info [o.e.x.s.a.r.TransportPutRoleAction] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updated role [fleet_enroll]
[00:01:46]                 │ info [o.e.x.s.a.u.TransportPutUserAction] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] updated user [fleet_enroll]
[00:01:48]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:49]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:50]               └-> should work
[00:01:50]                 └-> "before each" hook: global before each
[00:01:50]                 └-> "before each" hook: beforeSetupWithDockerRegistry
[00:01:50]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1601392901384064606] [.kibana_2/fNMu1X7KRXq2ih2P12GNew] update_mapping [_doc]
[00:01:54]                 └- ✖ fail: Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work
[00:01:54]                 │      Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
[00:01:54]                 │       at Assertion.assert (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:100:11)
[00:01:54]                 │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:227:8)
[00:01:54]                 │       at Assertion.be (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:69:22)
[00:01:54]                 │       at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
[00:01:54]                 │ 
[00:01:54]                 │ 

```

#### Stack Trace
```
Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
    at Assertion.assert (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/12/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/","number":"77911","commit":"f3357b8c53685f003f164b1dd6dfd17c7a263fd5"}]}
PIPELINE-->

jen-huang commented:
The failure on `Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'` could happen if we detect that the agent version is < 7.10.0 and thus send `CONFIG_CHANGE` instead. But the agent version used in the test suite is set to the Kibana version, which should be `7.10.0` on this branch. The test passes locally using this branch 🤔 Going to re-run CI to double check...

jen-huang commented:
@elasticmachine merge upstream

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/)
* Commit: 312cf73f11db1a1ecccd740788ccc94ecc1b8d56
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute xpack-kibana-ciGroup7](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/execution/node/633/log)
* [Execute xpack-kibana-ciGroup7](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/execution/node/762/log)

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/testReport)

<details><summary>X-Pack EPM API Integration Tests.x-pack/test/ingest_manager_api_integration/apis/fleet/agents/complete_flow·ts.Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/testReport//junit/X-Pack%20EPM%20API%20Integration%20Tests/x-pack_test_ingest_manager_api_integration_apis_fleet_agents_complete_flow·ts/Ingest_Manager_Endpoints_Fleet_Endpoints_fleet_agent_flow_should_work)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://github.com/elastic/kibana/issues/75241

[00:00:00]       │
[00:00:00]         └-: Ingest Manager Endpoints
[00:00:00]           └-> "before all" hook
[00:00:09]           └-: Fleet Endpoints
[00:00:09]             └-> "before all" hook
[00:01:31]             └-: fleet_agent_flow
[00:01:31]               └-> "before all" hook
[00:01:31]               └-> "before all" hook
[00:01:31]                 │ info [empty_kibana] Loading "mappings.json"
[00:01:31]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_1/j6Y3JvirQEadqAbBK8SBFA] deleting index
[00:01:31]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/CLiQzZuhRTmux8KbsTcrdw] deleting index
[00:01:31]                 │ info [empty_kibana] Deleted existing index [".kibana_2",".kibana_1"]
[00:01:31]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana] creating index, cause [api], templates [], shards [1]/[1]
[00:01:31]                 │ info [empty_kibana] Created index ".kibana"
[00:01:31]                 │ debg [empty_kibana] ".kibana" settings {"index":{"number_of_replicas":"1","number_of_shards":"1"}}
[00:01:31]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana/okeWxbahS9-8hXAOXgRXkg] update_mapping [_doc]
[00:01:31]                 │ debg Migrating saved objects
[00:01:31]                 │ proc [kibana]   log   [20:24:36.112] [info][savedobjects-service] Creating index .kibana_2.
[00:01:31]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2] creating index, cause [api], templates [], shards [1]/[1]
[00:01:31]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.kibana_2]
[00:01:31]                 │ proc [kibana]   log   [20:24:36.175] [info][savedobjects-service] Reindexing .kibana to .kibana_1
[00:01:31]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_1] creating index, cause [api], templates [], shards [1]/[1]
[00:01:31]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.kibana_1]
[00:01:31]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.tasks] creating index, cause [auto(task api)], templates [], shards [1]/[1]
[00:01:31]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.tasks]
[00:01:31]                 │ info [o.e.t.LoggingTaskListener] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] 5812 finished with response BulkByScrollResponse[took=2.2ms,timed_out=false,sliceId=null,updated=0,created=0,deleted=0,batches=0,versionConflicts=0,noops=0,retries=0,throttledUntil=0s,bulk_failures=[],search_failures=[]]
[00:01:31]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana/okeWxbahS9-8hXAOXgRXkg] deleting index
[00:01:31]                 │ proc [kibana]   log   [20:24:36.544] [info][savedobjects-service] Migrating .kibana_1 saved objects to .kibana_2
[00:01:31]                 │ proc [kibana]   log   [20:24:36.549] [info][savedobjects-service] Pointing alias .kibana to .kibana_2.
[00:01:31]                 │ proc [kibana]   log   [20:24:36.576] [info][savedobjects-service] Finished in 465ms.
[00:01:31]               └-> "before all" hook
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33932, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33930, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33938, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33940, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:31]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:31]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33944, url.original: /package/endpoint/0.13.1
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33948, url.original: /package/system/0.5.3
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33952, url.original: /package/endpoint/0.13.1/
[00:01:31]                 │ info [docker:registry] 2020/09/29 20:24:36 source.ip: 172.17.0.1:33956, url.original: /package/system/0.5.3/
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33964, url.original: /search?kibana.version=7.10.0
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33962, url.original: /search?kibana.version=7.10.0
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33972, url.original: /package/system/0.5.3
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33980, url.original: /package/endpoint/0.13.1
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33974, url.original: /package/endpoint/0.13.1
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33978, url.original: /package/system/0.5.3
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33990, url.original: /package/endpoint/0.13.1/
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33996, url.original: /package/system/0.5.3/
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33986, url.original: /package/system/0.5.3/
[00:01:32]                 │ info [docker:registry] 2020/09/29 20:24:37 source.ip: 172.17.0.1:33994, url.original: /package/endpoint/0.13.1/
[00:01:32]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:33]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:33]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:33]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:33]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34016, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34017, url.original: /package/system/0.5.3
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34022, url.original: /package/system/0.5.3/
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34030, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34028, url.original: /package/system/0.5.3
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34034, url.original: /package/system/0.5.3/
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34038, url.original: /package/system/0.5.3
[00:01:37]                 │ info [docker:registry] 2020/09/29 20:24:42 source.ip: 172.17.0.1:34042, url.original: /package/system/0.5.3/
[00:01:37]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:39]                 │ info [docker:registry] 2020/09/29 20:24:44 source.ip: 172.17.0.1:34052, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:39]                 │ info [docker:registry] 2020/09/29 20:24:44 source.ip: 172.17.0.1:34054, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:39]                 │ info [o.e.x.s.a.r.TransportPutRoleAction] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updated role [fleet_enroll]
[00:01:39]                 │ info [o.e.x.s.a.u.TransportPutUserAction] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updated user [fleet_enroll]
[00:01:41]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:42]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:43]               └-> should work
[00:01:43]                 └-> "before each" hook: global before each
[00:01:43]                 └-> "before each" hook: beforeSetupWithDockerRegistry
[00:01:43]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/D1NfG655TW-LU3lqtXk3PA] update_mapping [_doc]
[00:01:47]                 └- ✖ fail: Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work
[00:01:47]                 │      Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
[00:01:47]                 │       at Assertion.assert (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:100:11)
[00:01:47]                 │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:227:8)
[00:01:47]                 │       at Assertion.be (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:69:22)
[00:01:47]                 │       at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
[00:01:47]                 │ 
[00:01:47]                 │ 

```

#### Stack Trace
```
Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
    at Assertion.assert (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
```
</details>

---

<details><summary>X-Pack EPM API Integration Tests.x-pack/test/ingest_manager_api_integration/apis/fleet/agents/complete_flow·ts.Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/testReport//junit/X-Pack%20EPM%20API%20Integration%20Tests/x-pack_test_ingest_manager_api_integration_apis_fleet_agents_complete_flow·ts/Ingest_Manager_Endpoints_Fleet_Endpoints_fleet_agent_flow_should_work_2)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://github.com/elastic/kibana/issues/75241

[00:00:00]       │
[00:00:00]         └-: Ingest Manager Endpoints
[00:00:00]           └-> "before all" hook
[00:00:09]           └-: Fleet Endpoints
[00:00:09]             └-> "before all" hook
[00:01:34]             └-: fleet_agent_flow
[00:01:34]               └-> "before all" hook
[00:01:34]               └-> "before all" hook
[00:01:34]                 │ info [empty_kibana] Loading "mappings.json"
[00:01:34]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_1/U9Zh_ZjaSfiYlN6e0oCJuQ] deleting index
[00:01:34]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Pk0DPHLORLyObb5H7HnAeg] deleting index
[00:01:34]                 │ info [empty_kibana] Deleted existing index [".kibana_2",".kibana_1"]
[00:01:34]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana] creating index, cause [api], templates [], shards [1]/[1]
[00:01:34]                 │ info [empty_kibana] Created index ".kibana"
[00:01:34]                 │ debg [empty_kibana] ".kibana" settings {"index":{"number_of_replicas":"1","number_of_shards":"1"}}
[00:01:35]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana/sKi74rz8SUqVphWsk3oKqA] update_mapping [_doc]
[00:01:35]                 │ debg Migrating saved objects
[00:01:35]                 │ proc [kibana]   log   [19:36:18.953] [info][savedobjects-service] Creating index .kibana_2.
[00:01:35]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2] creating index, cause [api], templates [], shards [1]/[1]
[00:01:35]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.kibana_2]
[00:01:35]                 │ proc [kibana]   log   [19:36:19.023] [info][savedobjects-service] Reindexing .kibana to .kibana_1
[00:01:35]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_1] creating index, cause [api], templates [], shards [1]/[1]
[00:01:35]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.kibana_1]
[00:01:35]                 │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.tasks] creating index, cause [auto(task api)], templates [], shards [1]/[1]
[00:01:35]                 │ info [o.e.c.r.a.AllocationService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updating number_of_replicas to [0] for indices [.tasks]
[00:01:35]                 │ info [o.e.t.LoggingTaskListener] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] 5851 finished with response BulkByScrollResponse[took=2.2ms,timed_out=false,sliceId=null,updated=0,created=0,deleted=0,batches=0,versionConflicts=0,noops=0,retries=0,throttledUntil=0s,bulk_failures=[],search_failures=[]]
[00:01:35]                 │ info [o.e.c.m.MetadataDeleteIndexService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana/sKi74rz8SUqVphWsk3oKqA] deleting index
[00:01:35]                 │ proc [kibana]   log   [19:36:19.401] [info][savedobjects-service] Migrating .kibana_1 saved objects to .kibana_2
[00:01:35]                 │ proc [kibana]   log   [19:36:19.407] [info][savedobjects-service] Pointing alias .kibana to .kibana_2.
[00:01:35]                 │ proc [kibana]   log   [19:36:19.439] [info][savedobjects-service] Finished in 487ms.
[00:01:35]               └-> "before all" hook
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44720, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44718, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44728, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44726, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:35]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44736, url.original: /package/system/0.5.3
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44738, url.original: /package/endpoint/0.13.1
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44744, url.original: /package/system/0.5.3/
[00:01:35]                 │ info [docker:registry] 2020/09/29 19:36:19 source.ip: 172.17.0.1:44746, url.original: /package/endpoint/0.13.1/
[00:01:35]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44755, url.original: /search?kibana.version=7.10.0
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44754, url.original: /search?kibana.version=7.10.0
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44764, url.original: /package/endpoint/0.13.1
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44766, url.original: /package/system/0.5.3
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44772, url.original: /package/system/0.5.3
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44770, url.original: /package/endpoint/0.13.1
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44780, url.original: /package/system/0.5.3/
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44786, url.original: /package/system/0.5.3/
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44782, url.original: /package/endpoint/0.13.1/
[00:01:36]                 │ info [docker:registry] 2020/09/29 19:36:20 source.ip: 172.17.0.1:44788, url.original: /package/endpoint/0.13.1/
[00:01:36]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:37]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:37]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:37]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44896, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44894, url.original: /package/system/0.5.3
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44900, url.original: /package/system/0.5.3/
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44907, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44906, url.original: /package/system/0.5.3
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44912, url.original: /package/system/0.5.3/
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44916, url.original: /package/system/0.5.3
[00:01:41]                 │ info [docker:registry] 2020/09/29 19:36:25 source.ip: 172.17.0.1:44920, url.original: /package/system/0.5.3/
[00:01:41]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:42]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:43]                 │ info [docker:registry] 2020/09/29 19:36:27 source.ip: 172.17.0.1:44944, url.original: /search?package=system&internal=true&experimental=true&kibana.version=7.10.0
[00:01:43]                 │ info [docker:registry] 2020/09/29 19:36:27 source.ip: 172.17.0.1:44946, url.original: /search?package=endpoint&internal=true&experimental=true&kibana.version=7.10.0
[00:01:43]                 │ info [o.e.x.s.a.r.TransportPutRoleAction] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updated role [fleet_enroll]
[00:01:43]                 │ info [o.e.x.s.a.u.TransportPutUserAction] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] updated user [fleet_enroll]
[00:01:45]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:46]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:47]               └-> should work
[00:01:47]                 └-> "before each" hook: global before each
[00:01:47]                 └-> "before each" hook: beforeSetupWithDockerRegistry
[00:01:47]                 │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-centos-tests-xxl-1601403470441417540] [.kibana_2/Qn8y2InQRfW9MsMldgkq9Q] update_mapping [_doc]
[00:01:51]                 └- ✖ fail: Ingest Manager Endpoints Fleet Endpoints fleet_agent_flow should work
[00:01:51]                 │      Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
[00:01:51]                 │       at Assertion.assert (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:100:11)
[00:01:51]                 │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:227:8)
[00:01:51]                 │       at Assertion.be (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:69:22)
[00:01:51]                 │       at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
[00:01:51]                 │ 
[00:01:51]                 │ 

```

#### Stack Trace
```
Error: expected 'CONFIG_CHANGE' to equal 'POLICY_CHANGE'
    at Assertion.assert (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/23/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/ingest_manager_api_integration/apis/fleet/agents/complete_flow.ts:79:50)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #77911](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/) failed f3357b8c53685f003f164b1dd6dfd17c7a263fd5

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/","number":"77911","commit":"f3357b8c53685f003f164b1dd6dfd17c7a263fd5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/","number":"78018","commit":"312cf73f11db1a1ecccd740788ccc94ecc1b8d56"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78089/)
* Commit: 4b578eb38b959909a229d0f68a9728056f19a637
* This comment will update when the build is complete

### Failed CI Steps
* [Lint: eslint](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78089/execution/node/340/log)

### History
* :broken_heart: [Build #78018](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/) failed 312cf73f11db1a1ecccd740788ccc94ecc1b8d56
* :broken_heart: [Build #77911](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/) failed f3357b8c53685f003f164b1dd6dfd17c7a263fd5

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/","number":"77911","commit":"f3357b8c53685f003f164b1dd6dfd17c7a263fd5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/","number":"78018","commit":"312cf73f11db1a1ecccd740788ccc94ecc1b8d56"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78089/","number":"78089","commit":"4b578eb38b959909a229d0f68a9728056f19a637"}]}
PIPELINE-->

jen-huang commented:
The problem was that `semver.lt('7.10.0-SNAPSHOT', '7.10.0')` was returning `true`, causing the code to return `CONFIG_CHANGE` instead of `POLICY_CHANGE`. I've adjusted the condition and added tests in 676ce45 and will open a PR to cherry-pick this commit into master.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78162/)
* Commit: 0e545f753c9d96b699313eba6e9300a6dccdcabd

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #78089](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78089/) failed 4b578eb38b959909a229d0f68a9728056f19a637
* :broken_heart: [Build #78018](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/) failed 312cf73f11db1a1ecccd740788ccc94ecc1b8d56
* :broken_heart: [Build #77911](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/) failed f3357b8c53685f003f164b1dd6dfd17c7a263fd5

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/77911/","number":"77911","commit":"f3357b8c53685f003f164b1dd6dfd17c7a263fd5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78018/","number":"78018","commit":"312cf73f11db1a1ecccd740788ccc94ecc1b8d56"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78089/","number":"78089","commit":"4b578eb38b959909a229d0f68a9728056f19a637"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/78162/","number":"78162","commit":"0e545f753c9d96b699313eba6e9300a6dccdcabd"}]}
PIPELINE-->
